asserts/extkeypairmgr.go

```diff
@@ -114,11 +114,13 @@ func (em *ExternalKeypairManager) keyNames() ([]string, error) {
 	return knames.Names, nil
 }
 
+var errExternalKeypairNotFound = &keyNotFoundError{msg: "cannot find external key pair"}
+
 func (em *ExternalKeypairManager) findByName(name string) (PublicKey, *rsa.PublicKey, error) {
 	var k []byte
 	err := em.keyMgr("get-public-key", []string{"-f", "DER", "-k", name}, nil, &k)
 	if err != nil {
-		return nil, nil, fmt.Errorf("cannot find external key: %v", err)
+		return nil, nil, errExternalKeypairNotFound
```

Comment:
we do want to keep %v err here to help with debugging as this is calls something external

Reply:
That's a good point. Pushed an update